app/api/graphkb.js

```diff
@@ -72,9 +72,19 @@ const graphkbReviewStatus = async (graphkbToken, graphkbEntries) => {
   const {uri} = CONFIG.get('graphkb');
 
   const filters = {
-    '@rid': graphkbEntries.map(({kbStatementId}) => {
-      return kbStatementId;
-    }),
+    '@rid': graphkbEntries
+        .reduce((filtered, {kbStatementId}) => {
+            if (kbStatementId.match(/^#\d+:\d+$/)) {
+                filtered.push(kbStatementId);
+            }
+            return filtered;
+        }, []);
+      .filter(({kbStatementId}) => {
```

Comment:
You need to remove the .filter and .map operations as well.

Reply:
Oops, I tried using the github accept suggestion thing right in the gui. I won't be doing that again...